Rewrite the 'PivotGrid - Virtual Scrolling' demo description (#1141)
DevExpress/devextreme-demos: JSDemos/Demos/PivotGrid/VirtualScrolling/description.md

If the PivotGrid is bound to a large dataset, you can enable the virtual scrolling mode to optimize data load time and reduce resource consumption. In this mode, the PivotGrid loads only the cells that are in the viewport. When cells leave the viewport, the PivotGrid removes them from the DOM.

> To allow users to scroll the PivotGrid virtually, set the scrolling.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.

Reviewer: ```suggestion
To allow users to scroll the PivotGrid virtually, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
```

> To allow users to scroll the PivotGrid virtually, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.

Reviewer 2: I don't know about the phrase "scroll the control virtually". You are really (actually) scrolling it. I would just stick the term "Virtual Scrolling", which is the name of the demo. But then I don't want to repeat the words "mode" and "virtual" too many times. So maybe something simple like this:

```suggestion
To enable this mode, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
```

> If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load only the cells that are in the viewport. This allows you to optimize data load time and reduces resources consumption. When users scroll the PivotGrid, it loads cells to be displayed in the viewport and removes other cells from memory.

Reviewer 3: If the component removes the cell from the DOM, then:
```suggestion
If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load cells of the current viewport only. This allows you to optimize data load time and reduce resources consumption. When cells leave the viewport, the PivotGrid removes them from the DOM.
```

Author: >load cells of the current viewport only.

Cells are separate UI elements that are within another UI element - viewport. 

```suggestion
If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load only cells that are in the viewport only. This allows you to optimize data load time and reduce resources consumption. When cells leave the viewport, the PivotGrid removes them from the DOM.
```

Reviewer: > resources consumption

Even though we are talking about multiple resources, the compound noun should have "resource" in singular form.

I also changed the structure a bit. Your version starts with information about how the virtual scrolling works ("load only the cells that are in the viewport"), then give a bit of into about about how it can be helpful ("allows you to optimize data load time and reduce resources consumption"), then goes back to how it works ("When cells leave the viewport, the PivotGrid removes them from the DOM."). I put the two similar pieces together in my version.

```suggestion
If the PivotGrid is bound to a large dataset, you can enable the virtual scrolling mode to optimize data load time and reduce resource consumption. In this mode, the PivotGrid loads only the cells that are in the viewport. When cells leave the viewport, the PivotGrid removes them from the DOM.
```

> To allow users to scroll the PivotGrid virtually, set the scrolling.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to "virtual".

Reviewer 3: ```suggestion
To allow users to scroll the PivotGrid virtually, set the scrolling.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
```

> In this demo, the PivotGrid contains approximately 4000 cells. To see virtual scrolling in action, scroll the PivotGrid horizontally.

Reviewer 3: ```suggestion
In this demo, the PivotGrid stores approximately 4000 cells locally. To see virtual scrolling in action, scroll the PivotGrid horizontally.
```

Author: The pivot grid can contain cells since they are UI elements, but "store" implies that we talk about data. We can say that it contains records instead:

UPD: I googled the term 'record' in computer science and found out that it relates to a row or a set of fields. Changed it to 'values'.
```suggestion
In this demo, the PivotGrid stores approximately 4000 values locally. To see virtual scrolling in action, scroll the PivotGrid horizontally.
```

> To see virtual scrolling when the PivotGrid is bound to a remote storage, refer to the [Remote Virtual Scrolling](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/RemoteVirtualScrolling) demo.

Reviewer: Remote Virtual Scrolling is the next demo in the list. I believe there's no reason to give backlinks in this case.

> This demo shows the PivotGrid bound to the local storage. To see virtual scrolling when cells display data from a server, refer to the [Remote Virtual Scrolling](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/RemoteVirtualScrolling) demo.

Reviewer 3: ```suggestion
To see virtual scrolling when the PivotGrid is bound to a remote storage, refer to the [Remote Virtual Scrolling](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/RemoteVirtualScrolling) demo.
```
